src/test/java/org/threeten/extra/chrono/TestInternationalFixedChronology.java

```diff
@@ -0,0 +1,1191 @@
+    }
+
+    @Test(dataProvider = "samples")
+    public void test_LocalDate_until_InternationalFixedDate(final InternationalFixedDate date, final LocalDate iso) {
+        assertEquals(iso.until(date), Period.ZERO);
+    }
+
+    @Test(dataProvider = "samples")
+    public void test_Chronology_date_Temporal(final InternationalFixedDate date, final LocalDate iso) {
+        assertEquals(InternationalFixedChronology.INSTANCE.date(iso), date);
+    }
+
+    @Test(dataProvider = "samples")
+    public void test_plusDays(final InternationalFixedDate date, final LocalDate iso) {
+        assertEquals(LocalDate.from(date.plus(0, ChronoUnit.DAYS)), iso);
+        assertEquals(LocalDate.from(date.plus(1, ChronoUnit.DAYS)), iso.plusDays(1));
+        assertEquals(LocalDate.from(date.plus(35, ChronoUnit.DAYS)), iso.plusDays(35));
+    }
+
+    @Test(dataProvider = "samples")
+    public void test_minusDays(final InternationalFixedDate date, final LocalDate iso) {
+        assertEquals(LocalDate.from(date.minus(0, ChronoUnit.DAYS)), iso);
+        assertEquals(LocalDate.from(date.minus(-1, ChronoUnit.DAYS)), iso.minusDays(-1));
+        assertEquals(LocalDate.from(date.minus(-60, ChronoUnit.DAYS)), iso.minusDays(-60));
+    }
+
+    @Test(dataProvider = "samples")
+    public void test_until_DAYS(final InternationalFixedDate date, final LocalDate iso) {
+        assertEquals(date.until(iso.plusDays(0), ChronoUnit.DAYS), 0);
+        assertEquals(date.until(iso.plusDays(1), ChronoUnit.DAYS), 1);
+        assertEquals(date.until(iso.plusDays(35), ChronoUnit.DAYS), 35);
+    }
+
+    @DataProvider(name = "badDates")
+    Object[][] data_badDates() {
+        return new Object[][] {
+                {    0, 1, 1 },
+
+                { 1900, 0, 0 },
+                { 1900, -1, 1 },
+                { 1900, 0, 1 },
+                { 1900, 15, 1 },
+                { 1900, 16, 1 },
+
+                { 1900, 1, -1 },
+                { 1900, 1, 0 },
+                { 1900, 1, 29 },
+
+                { 1900, 13, -1 },
+                { 1900, 13, 0 },
+                { 1900, 13, 88 },
+                { 1900, 14, -1 },
+                { 1900, 14, 0 },
+                { 1900, 14, 29 },
+                { 1900, 14, 30 },
+
+                { 1898, 13, -1 },
+                { 1898, 13, 0 },
+                { 1898, 14, 29 },
+                { 1898, 14, 30 },
+                { 1898, 14, 1 },
+                { 1898, 14, 2 },
+
+                { 1900, 14, -1 },
+                { 1900, 14, 0 },
+                { 1900, 14, 29 },
+
+                { 1900, 2, 29 },
+                { 1900, 3, 29 },
+                { 1900, 4, 29 },
+                { 1900, 5, 29 },
+                { 1900, 6, 29 },
+                { 1900, 7, 29 },
+                { 1900, 8, 29 },
+                { 1900, 9, 29 },
+                { 1900, 10, 29 },
+                { 1900, 11, 29 },
+                { 1900, 12, 29 },
```

Comment:
Missing 1900, 13, 29.

Most of the rest of the data in this test needs to be reviewed, as a bunch of it was dealing with existence/absence of the Pax leap-week.  You're also not testing for things like invalid leap years here, which you should be.